docs/developer-guide.md

```diff
@@ -0,0 +1,88 @@
+# Developers' guide
+
+This page describes how to use pipenv library and pipenv commands when you are developing on this repository.
```

Comment:
maybe edit this overview to reflect that this page is not only about pipenv but also general development including roles and integration tests

Reply:
Yes, you are right, that was a leftover since it was created as developer's guide to pipenv